### ISSUE TITLE
Editorial color mode doesn't correspond with the current color mode when toggling "dark mode"

### ISSUE THREAD

**Issue by VioAlbert** [ia-toki/judgels#406]
There seems to be a bug when toggling "dark mode"

Steps to recreate:
1. Switch to dark mode
2. Choose a problem
2. Open editorial
3. Switch to light mode
4. Open editorial again

Expected outcome:
The color of the panel should correspond with the current mode.

![Screenshot (572)](https://user-images.githubusercontent.com/36184798/132122204-385bbe06-16f3-4abb-a12a-2f45316b0886.png)

**Comment by fushar** [comment]
Yes, this is a known issue with the current versions of Preact and Blueprint. The only way to fix this is to refresh the page. Will take a look into this when new versions come out.
